src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/create_index_pattern_wizard.js

```diff
@@ -52,20 +53,35 @@ export class CreateIndexPatternWizard extends Component {
       step: 1,
       indexPattern: '',
       allIndices: [],
+      remoteClustersExist: false,
       isInitiallyLoadingIndices: true,
       isIncludingSystemIndices: false,
     };
   }
 
   async componentWillMount() {
-    this.fetchIndices();
+    this.fetchData();
   }
 
-  fetchIndices = async () => {
-    this.setState({ allIndices: [], isInitiallyLoadingIndices: true });
+  fetchData = async () => {
     const { services } = this.props;
-    const allIndices = await ensureMinimumTime(getIndices(services.es, this.indexPatternCreationType, `*`, MAX_SEARCH_SIZE)); //
-    this.setState({ allIndices, isInitiallyLoadingIndices: false });
+
+    this.setState({
+      allIndices: [],
+      isInitiallyLoadingIndices: true,
+      remoteClustersExist: false
+    });
+
+    const [allIndices, remoteClusters] = await ensureMinimumTime([
```

Comment:
I ran into a scenario where I had my remote cluster mis-configured, resulting in a 500 timeout error, but the UI remained stuck on loading. I think this area of the code could benefit from try/catch to handle ES errors from either of the requests, but I won't block the PR on that because the previous version didn't have good error handling either